src/database/migrations/z1631138691-create-notification.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+// eslint-disable-next-line no-undef
+module.exports = {
+    async up(queryInterface, Sequelize) {
+        return queryInterface.createTable('app_notification', {
+            id: {
+                type: Sequelize.INTEGER,
+                autoIncrement: true,
+                primaryKey: true,
+            },
+            address: {
+                type: Sequelize.STRING(44),
+                references: {
+                    model: 'user',
+                    key: 'address',
```

Comment:
TBH, I've been thinking that anything that is not related to on-chain activity, would be linked to the user id, instead of the address. The address is a 44 chars string. The user id is an integer index. I see many advantages. What's your opinion?

Reply:
it is a good point. 
I was searching about it and really is better (more efficient and simpler) to do comparisons and joins by an integer (ID) than a string.
so, we can have faster queries, especially the queries with the `includes`